### PR TITLE
Finished 100-105

### PR DESCRIPTION
**Pull request by prateeky2806**
Hi, I have modified the sample f_100-f_105 to improve them. Also, there were some empty files 141, 144, 155 and 158 which have been deleted from the raw folder. 

Can you please check if these changes look good to you?

### CODE REVIEW

File: data/raw/f_102_armel.py
Comment (terryyz) [inline data/raw/f_102_armel.py:1]: `string` is currently not being used in the main function. Two choices for the refinement are:
- Putting line 7 to the main function (preferred)
- Removing `string` requirements from the docstring

**Review by terryyz — COMMENTED** [review]
Thanks @prateeky2806! 

A quick note on checking the number of libs used in each main function:
- You can utilize the libs keys in the processed "data/open-eval.jsonl" to check if at least 2 libs are used for each sample.

**Comment by prateeky2806** [comment]
> Thanks @prateeky2806!
> 
> A quick note on checking the number of libs used in each main function:
> 
> * You can utilize the libs keys in the processed "data/open-eval.jsonl" to check if at least 2 libs are used for each sample.

Yes, I am already checking the number of libraries used in each of the files in the raw folder, then why do I need to check it in the open-eval.jsonl?

**Comment by terryyz** [comment]
> > Thanks @prateeky2806!
> > A quick note on checking the number of libs used in each main function:
> > 
> > * You can utilize the libs keys in the processed "data/open-eval.jsonl" to check if at least 2 libs are used for each sample.
> 
> Yes, I am already checking the number of libraries used in each of the files in the raw folder, then why do I need to check it in the open-eval.jsonl?

Oh, I mean that the parse.py will help you check if you don't want to spend too much time looking into the used libraries. In addition, it will be helpful to double-check any samples missing the criteria 😃

File: data/raw/f_119_armel.py [hunk 1]
```diff
@@ -19,17 +19,20 @@ def f_119(timestamps):
     Returns:
     - pandas.DataFrame: A pandas DataFrame containing the original Unix timestamps and the converted datetime objects.
     - Axes: The Axes object of the histogram plot. The histogram will have 10 bins by default, representing the distribution of the datetime objects.
+    - ValueError("Input list of timestamps is empty"): If the list of timestamps is empty.
```
Comment (terryyz) [inline data/raw/f_119_armel.py:24]: Please put any exceptions under a section called "Raises:"

**Comment by prateeky2806** [comment]
Hi @terryyz, I have finished 100-150. 

Thanks.

**Comment by terryyz** [comment]
Thanks @prateeky2806! Could you please update `script/parse.py` and `script/run.sh` to the latest version in your main branch, and see if there are any issues? The ones in your main branch are a bit out of the time. Please note that you need to change `NAMES` to `armel` and add the specific range of your refined split. Otherwise, the whole `armel` split will be tested.

**Comment by terryyz** [comment]
Close for now, as I've finished debugging them. I'll add them in my next commit.

**Pull request closed without merging**